### DISCUSSION TITLE
How many points do you have?

### DISCUSSION THREAD

**Discussion by ghost** [IBAX-io/go-ibax#1084] [Ecolibs]
Hi, how many points did you score so far? 
  Me:25k

  You:?

**Comment by Skips1** [comment]
26 k bro, 1 k task came, do it..

**Reply by ghost** [reply]
Thank you.

**Comment by Skips1** [comment]
All will be revealed soon

**Comment by yasirozturk01** [comment]
All will be revealed soon
